src/components/common/pills/pill-items-child.styles.tsx

```diff
@@ -10,9 +10,12 @@ export const Wrapper = styled.div`
 `;
 
 export const DroppableWrapper = styled.div<IDroppableWrapperProps>`
-    border: 1px dashed ${Color.Primary};
-    border-radius: 0.25rem;
-    background: ${Color.Accent.Light[5]};
+    ${Border.Util["dashed-default"]({
+        thickness: Border["width-010"],
+        colour: Colour["border-focus-strong"],
+    })};
+    border-radius: ${Radius.xs};
+    background: ${Colour["bg-primary-subtlest"]};
```

Comment:
```suggestion
    background-color: ${Colour["bg-primary-subtlest"]};
```

the border util uses `background-image` under the hood, so we need to specifically target the background color property to avoid overwriting it